client/src/App.js

```diff
@@ -9,15 +9,17 @@ class App extends Component {
     super(props);
     this.state = {
       step: 1,
-      isWorking: -1
+      isWorking: -1,
+      erorr: null
```

Comment:
there is a typo here
```suggestion
      error: null
```